### PR TITLE
feat: command delete variables

### PR DESCRIPTION
**Pull request by maxwelbm**
- fix: update command deploy
- feat: command delete variables

### CODE REVIEW

File: pkg/cmd/deploy/requests.go [hunk 1]
```diff
@@ -177,6 +192,16 @@ func (cmd *DeployCmd) doOrigin(client *apiapp.Client, clientOrigin *apiori.Clien
 			return err
 		}
 		logger.FInfo(cmd.F.IOStreams.Out, msg.RulesEngineSuccessful)
+	} else {
+		reqObjectStorageOrigin := apiori.UpdateRequest{}
+		reqObjectStorageOrigin.Prefix = &conf.Prefix
+
+		_, err := clientOrigin.Update(ctx, conf.Application.ID, conf.Origin.StorageOriginKey, &reqObjectStorageOrigin)
+		if err != nil {
+			logger.Debug("Error while update origin of type object storage", zap.Any("Error", err))
```
Comment (PatrickMenoti) [inline pkg/cmd/deploy/requests.go:201]: Error while updating origin of type object storage

File: pkg/cmd/deploy/requests.go [hunk 2]
```diff
@@ -112,10 +125,11 @@ func (cmd *DeployCmd) doOrigin(client *apiapp.Client, clientOrigin *apiori.Clien
 	if conf.Template == "javascript" || conf.Template == "typescript" {
 		return nil
 	}
+
 	var addresses []string
 	var DefaultOrigin = [1]string{"httpbin.org"}
 
-	if conf.Mode == "compute" {
+	if strings.ToLower(conf.Mode) == "compute" {
```
Comment (PatrickMenoti) [inline pkg/cmd/deploy/requests.go:132]: Technically we do not need this, because when we save them we already should be setting them to lowercase 🤔

Comment (jotanarciso) [reply pkg/cmd/deploy/requests.go:132]: I believe it is a valid validation, do not trust the user (it can change and will 😂)

Comment (PatrickMenoti) [reply pkg/cmd/deploy/requests.go:132]: @jotanarciso makes sense 😆

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**